src/js/components/RadioButtonGroup/RadioButtonGroup.js

```diff
@@ -83,7 +84,12 @@ const RadioButtonGroup = forwardRef(
       // Chrome behaves differently in that focus is given to radio buttons
       // when the user selects one, unlike Safari and Firefox.
       setTimeout(() => {
-        setFocus(true);
+        // In case the focus needs to be trapped inside the input instead of the
+        // Box wrapper
+        if (trapFocus) {
```

Comment:
Hey Shimi! Thanks for the proposal. I understand the use case, but I'm trying to think of a better way to handle it. A couple reactions:
- The reuse of `trapFocus` as a prop name doesn't quite feel applicable here because in the case of Drop/Layer, trapFocus indicates that something should behave like a modal and not allow tab stops to continue to anywhere else on the page. That is a bit different from the intent here to place the focus on an individual radiobutton.
- `setFocus` in the code is just a boolean to track if focus is set one a radiobutton but it doesn't actually perform the focus behavior.

I don't have a suggestion at the moment for a better path, but just wanted to drop my reactions and let you know that I'm thinking on it.

Reply:
Thanks for the feedback, appreciate your time and thoughts on this.
I agree it is not trivial. I sent the initial proposal to get the convo started so I appreciate you chiming in.

Here are some follow-up thoughts.
* I had a similar thought after filing the PR regarding the name of the prop. A name like 'setFirstInputFocus', 'defaultFocusToFirst', 'focusFirstInGroup' might be more self-explanatory - let me know if you like any of those, or whether we should branch off on of them.
* Yes I know, I'll need to work on it further, this is why the PR is still in Draft. I was seeking initial thoughts from the team before I continue to invest forward in the solution.

Reply:
Thanks for working on this, I'm wondering if a potential workaround you team could use is before setting the focus on `input.current` check if `role="radiogroup"`. If so instead of focusing `input.current` do something like this:
```
const inputEl = input.current?.querySelectorAll('input[type=radio]')[0];
inputEl.focus();
```